table.gs

```diff
@@ -415,12 +415,12 @@ Table.prototype.add = function(input_item) {
  */
 Table.prototype.sortBy = function(key, ascending) {
 
-  this.items.sort(function(a, b){
-
-    if (a.getFieldValue(key) instanceof Date) {
-      var keyA = a.getFieldValue(key).getTime();
-      var keyB = b.getFieldValue(key).getTime();
-
+  this.items.sort(function(a, b) {
+    if (!isNaN(Date.parse(a.getFieldValue(key))) && !isNaN(Date.parse(b.getFieldValue(key)))) {
```

Comment:
It's difficult to read...

i'd do: 
```
var valueA = a.getFieldValue(key);
var valueB = b.getFieldValue(key);
```

and change following code accordingly